assets/queries/cloudFormation/disable_CDN/query.rego

```diff
@@ -0,0 +1,17 @@
+package Cx
+
+CxPolicy [result] {
+  resource := input.document[i].Resources[name]
+  properties := input.document[i].Resources[name].Properties[j]
+  distributionConfig := properties.Enabled
+  expectedvalue := "false"
+  distributionConfig == expectedvalue
+  object.get(properties, "Origins", "undefined") == "undefined"
```

Comment:
What if only one of these checks is true, shouldn't also be caught?

Reply:
This still hasn't been addressed. The check finds a failure when Not (Enabled Is True and Origins Is Defined), meaning, Enabled Is Not True or Origins Is Not Defined